src/shared/components/render-lines/render-lines.tsx

```diff
@@ -0,0 +1,12 @@
+export const RenderLines = (text: string) => {
+  return (
+    <p>
+      {text.split("\n").map((line, idx) => (
+        <>
+          {line}
+          {idx !== text.split("\n").length - 1 && <br />}
+        </>
+      ))}
+    </p>
+  );
+};
```

Comment:
RenderLines가 legal-notice의 줄바꿈만 처리해주는 용도인가요??
그렇다면 \n 기준으로 split 해서 <br />를 넣어주지 않고 CSS `white-space: pre-wrap`으로 정리해도 될 것 같아요.
이렇게 하면 렌더링 때마다 text.split("\n")를 여러 번 호출하지 않아도 될것 같네요!

```suggestion
export const RenderLines = (text: string) => {
  return <p style={{ whiteSpace: "pre-wrap" }}>{text}</p>;
};
```
그리고 스타일 파일은 따로 빼고 폰트 스타일과 색 지정하면 될것 같아요

Reply:
완전 맞다고 생각합니다 👍👍